umm-spec-lib/src/cmr/umm_spec/validation/platform.clj

```diff
@@ -34,24 +37,46 @@
    with the model number. So the short name is not specific enough to identify
    specific platform instances. Therefore we are adding Characteristic names and values
    so that specific aircraft tail numbers, or boat IDS, etc. can be specified with the
-   short name."
+   short name. The Unicode Character 'SYMBOL FOR GROUP SEPARATOR' \u241D character is added as a delimiter between the fields. This is used
+   in the getErrorString parameter-parser to parse out the different fields, so that a
+   meaningful error string can be suppplied back to the user."
   [values unique-fields]
   (let [{:keys [ShortName Characteristics Name Value]} unique-fields]
     (map (fn [value]
            (apply str
                  (ShortName value)
-                 (map #(str (Name %) (Value %)) (Characteristics value))))
+                 "\u241D"
+                 (map #(str (Name %) "\u241D" (Value %)) (Characteristics value))))
          values)))
 
+(defn- getErrorString
+  "Creates the error string for platforms depending on which elements (ShortName, Characteristic/Name, Characteristic/Value) are present.
+   The element order is ShortName, Name, Value. The values are separated by the Unicode Character 'SYMBOL FOR GROUP SEPARATOR' \u241D character."
+  [duplicate-name]
+  (when duplicate-name
+    (let [platform-elements (clojure.string/split duplicate-name #"\u241D")
```

Comment:
Our style is to require clojure.string :as string and then use the alias here `string/split`.